http-gateway/pkg/server/gateway_integration_test.go

```diff
@@ -126,10 +126,27 @@ func doMessage(port int, topic string, body io.Reader, headerKV ...string) *http
 
 func post(port int, path string, body io.Reader, headerKV ...string) *http.Response {
 	client := http.Client{}
-	req, _ := http.NewRequest("POST", fmt.Sprintf("http://localhost:%v%v", port, path), body)
+	req, err := http.NewRequest("POST", fmt.Sprintf("http://localhost:%v%v", port, path), body)
+	if err != nil {
```

Comment:
As @ericbottard was intimating, this block should be replaced by `Expect(err).NotTo(HaveOccurred())`, throughout.